### PR TITLE
Documentation Improvements

### PR DESCRIPTION
**Pull request by zeelot**
Making a pull request for the sake of tracking these changes. I'll be merging it in after fixing a group of documentation pages (accuracy and clarity).

Revamp Ticket: http://dev.kohanaframework.org/issues/4723
Cookie Salt Ticket: http://dev.kohanaframework.org/issues/4644

### CODE REVIEW

File: guide/kohana/install.md [hunk 1]
```diff
@@ -1,45 +1,64 @@
...
+
+ - Make sure the `application/cache` and `application/logs` directories are writable by the web server.
+~~~
+sudo chmod -R a+rwx application/cache
+sudo chmod -R a+rwx application/logs
+~~~
+
+ - Define a salt for the `Config` class.
+~~~
+Cookie::$salt = 'd131dd02c5e6eec4693d9a0698aff95c2fcab58712467eab4004583eb8fb7f89'
```
Comment (rjd22) [inline guide/kohana/install.md]: I would change the string to [really-long-cookie-salt-here] and link then to http://www.random.org/strings/ to generate their own random string.

**Comment by zeelot** [comment]
Made that change, thanks.

**Comment by rjd22** [comment]
I think it's good enough to merged in. Don't you agree?

**Comment by zeelot** [comment]
I want to improve many other pages as part of this, including removing the silly "Getting Started" submenu as that is just hiding pages under an arbitrary category.

**Pull request merged**